### ISSUE TITLE
Should I get the k by sorted student's logits or teacher's logits?

### ISSUE THREAD

**Issue by Geek-lixiang** [WisonZ/GKD#3]
Should I get the k by sorted student's logits or teacher's logits?  there are two loss in your code.

**Comment by WisonZ** [comment]
We provide codes for both situations in [gkd_adaptive.py](https://github.com/WisonZ/GKD/blob/main/gkd_adaptive.py). In our experiments, we found that getting k from student is better.
